android/gradle/dependencies.gradle

```diff
@@ -16,7 +16,7 @@
 
 def versions = [
     dagger: "2.22.1",
-    support: '25.1.1',
+    support: '1.0.0',
```

Comment:
Can you separate out the versions? The main advantage with androidx is that artifacts can have their own versions.

Reply:
done